.circleci/config.yml

```diff
@@ -34,7 +34,19 @@ jobs:
       - deploy:
           name: Deploy
           command: |
+            # Do not release on Broad Holidays
+            CURRDATE=$(date +%m/%d/%Y)
+             array=("03/12/2021" "4/19/2021" "5/28/2021" "5/31/2021" "7/2/2021" "7/5/2021"
```

Comment:
It looks like there's an extra space character on this line compared to the `CURDATE` line, was that deliberate?

Reply:
Not intentional, have removed it